introduction.md

```diff
@@ -2,4 +2,6 @@ Hello. I am Igor.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?